cupyx/_ufunc_config.py

```diff
@@ -1,28 +1,32 @@
 import contextlib
 import threading
 
-
 config = threading.local()
 config.divide = None
 config.over = None
 config.under = None
 config.invalid = None
+# In favor of performance, the `devInfo` input/output from cuSOLVER routine
+# calls that is necessary to check the validity of the other outputs, are
+# ignored, as D2H copy incurring device synchronizations would otherwise be
+# required.
+config.linalg = 'ignore'
 
 
 @contextlib.contextmanager
-def errstate(*, divide=None, over=None, under=None, invalid=None):
+def errstate(*, divide=None, over=None, under=None, invalid=None, linalg=None):
```

Comment:
I guess we should document all these arguments in all these functions, in a separate PR.